core/tests/linter_tests/invalid_constant.constants.ajs.ts

```diff
@@ -0,0 +1,42 @@
+// Copyright 2019 The Oppia Authors. All Rights Reserved.
```

Comment:
We also have a file `invalid_constant.ajs.ts` and this is `invalid_constant.constants.ajs.ts`. Is this intentional?

Reply:
This is intentional